src/rust/tests/read.rs

```diff
@@ -328,7 +317,7 @@ fn reader_type() -> Result<()> {
     fn get_feature_reader<R: Read + Seek>(
         reader: R,
     ) -> Result<FeatureIter<R, reader_trait::Seekable>> {
-        FgbReader::open(reader)?.select_all()
+        Ok(FgbReader::open(reader)?.select_all()?)
```

Comment:
This is the same as before, right? Is that better style?

Reply:
`?` here might call an `Into` trait, whereas simply returning a `Result` might not work in case the error type is different